### PR TITLE
Jbrowse: python3 + new options

### PR DESCRIPTION
**Pull request by abretaud**
A tiny fix for a problem with the jbrowse script and python3...

### CODE REVIEW

**Comment by abretaud** [comment]
Oh, hang on before merging, there are other python3 problems. Looks like there were some changes in the conda deps, I'll fix this tomorrow

File: tools/jbrowse/jbrowse.py [hunk 1]
```diff
@@ -370,7 +370,7 @@ def _add_json(self, json_data):
         self.subprocess_check_call(cmd)
 
     def _add_track_json(self, json_data):
-        if len(json_data.keys()) == 0:
+        if len(list(json_data.keys())) == 0:
```
Comment (nsoranzo) [inline tools/jbrowse/jbrowse.py]: That's not needed: https://docs.python.org/3/library/stdtypes.html#dictionary-view-objects

The simplest code is `len(json_data)` anyway.

Comment (abretaud) [inline tools/jbrowse/jbrowse.py]: Oh yes, I blindly trusted 2to3 on this

**Comment by abretaud** [comment]
I think it's ready for merge now, it worked fine with some real data tonight

**Comment by abretaud** [comment]
As I needed some additional options, I just added them to this PR.
They are described there: http://gmod.org/wiki/JBrowse_FAQ#My_CanvasFeatures_don.27t_show_up_with_subfeatures.2C_why_not.3F

**Comment by nsoranzo** [comment]
Python3 support looks good, I'll leave the rest to @erasche.

**Comment by bgruening** [comment]
Let's leave @erasche rest a little bit and merge it directly. Looks great to me!

**Pull request merged**